python/src/trezorlib/cli/__init__.py

```diff
@@ -307,6 +307,9 @@ def session_context(
             sys.exit(1)
         except exceptions.FailedSessionResumption:
             sys.exit(1)
+        except exceptions.DerivationOnUninitaizedDeviceError:
+            click.echo("Device is not initialized.")
+            sys.exit(1)
         except Exception:
             click.echo("Failed to find a Trezor device.")
```

Comment:
we definitely want to emit the exception in the log
maybe it's also a good idea to say `f"(error was: {e}"` or something?